example.py

```diff
@@ -0,0 +1,12 @@
+from phonon.connections import connect
+connect(hosts=['localhost'])
+
+from phonon.fields import *
+from phonon.model import Model
+
+class Foo(Model):
+    a = SumField()
```

Comment:
Ah, I didn't update the example w/ the code as it matured. I'll just remove this, it should be in the README if anywhere.